ui/src/components/RollupRulesTable.js

```diff
@@ -38,7 +38,7 @@ function TargetPreview({target}) {
         <div>{target.policies.join(' ')}</div>
       </div>
       <div className="pb1">
-        <label className="bold pb1">Tags</label>
+        <label className="bold pb1">GroupBy Tags</label>
```

Comment:
Let's still call this `Rollup Tags` for now and see whether the user finds it confusing.

Reply:
sg